src/cljdoc/render/layout.clj

```diff
@@ -78,11 +78,50 @@
    {:style {:top (str TOP-BAR-HEIGHT "px") :left "16rem"}} ; CSS HACK
    contents])
 
+(defn meta-info-icon []
+  [:img#meta-icon.ma3
+   {:src "/meta-info-icon.svg"
+    :alt "meta info icon"
+    :width "60px"
+    :style {:position "fixed"
+            :right 0
+            :bottom 0
+            :background "white"}}])
+
+(defn meta-element [description link]
```

Comment:
Since this is only used once I think it could also be moved into `meta-info-dialog`.